plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -67,6 +70,10 @@ export function createPublishGithubAction(options: {
             type: 'string',
             enum: ['private', 'public', 'internal'],
           },
+          collaborators: {
+            title: 'Collaborators',
```

Comment:
Think it makes sense to type this like you have done in your example in the PR rather than it just being `object`?

Reply:
Good point, I'll do that now.

Reply:
Added more granular definition to the object. 👍